src/store.js

```diff
@@ -0,0 +1,48 @@
+import { defineStore } from "pinia";
+import { fetchBreeds } from "./data.js";
+export const useStore = defineStore("store", {
+  state: () => {
+    return {
+      breedsList: [],
+      searchList: [],
+      searchQuery: "",
+    };
+  },
+  getters: {
+    displayedBreeds(state) {
+      if (this.searchQuery) {
+        return this.searchList;
+      } else {
+        return this.breedsList;
+      }
+    },
+  },
+  actions: {
+    fetchData() {
+      // this.isLoading = true;
+      // this.message = "Data is loading...";
+
+      fetchBreeds()

```

Comment:
mogoče lahko zdj vse iz `data.js` prestaviš sem v store.. da nimaš preveč ločene stvari 👍 

lahko je to nasleden PR ko tega spimpava

Reply:
Bom v naslednjem PR to naredu :D